[INFO][21-03-06][class com.yi.dao.UserDaoTest]info:进入了testlog4j
[DEBUG][21-03-06][class com.yi.dao.UserDaoTest]debug:进入了testLog4j
[ERROR][21-03-06][class com.yi.dao.UserDaoTest]error:进入了testLog4j
[INFO][21-03-06][class com.yi.dao.UserDaoTest]info:进入了testlog4j
[DEBUG][21-03-06][class com.yi.dao.UserDaoTest]debug:进入了testLog4j
[ERROR][21-03-06][class com.yi.dao.UserDaoTest]error:进入了testLog4j
[DEBUG][21-03-06][org.apache.ibatis.logging.LogFactory]Logging initialized using 'class org.apache.ibatis.logging.log4j.Log4jImpl' adapter.
[DEBUG][21-03-06][org.apache.ibatis.logging.LogFactory]Logging initialized using 'class org.apache.ibatis.logging.log4j.Log4jImpl' adapter.
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]PooledDataSource forcefully closed/removed all connections.
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]PooledDataSource forcefully closed/removed all connections.
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]PooledDataSource forcefully closed/removed all connections.
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]PooledDataSource forcefully closed/removed all connections.
[DEBUG][21-03-06][org.apache.ibatis.transaction.jdbc.JdbcTransaction]Opening JDBC Connection
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]Created connection 520022247.
[DEBUG][21-03-06][org.apache.ibatis.transaction.jdbc.JdbcTransaction]Setting autocommit to false on JDBC Connection [com.mysql.jdbc.JDBC4Connection@1efee8e7]
[DEBUG][21-03-06][com.yi.dao.UserMapper.getUserById]==>  Preparing: select * from mybatis.user where id = ? 
[DEBUG][21-03-06][com.yi.dao.UserMapper.getUserById]==> Parameters: 3(Integer)
[DEBUG][21-03-06][com.yi.dao.UserMapper.getUserById]<==      Total: 1
[DEBUG][21-03-06][org.apache.ibatis.transaction.jdbc.JdbcTransaction]Resetting autocommit to true on JDBC Connection [com.mysql.jdbc.JDBC4Connection@1efee8e7]
[DEBUG][21-03-06][org.apache.ibatis.transaction.jdbc.JdbcTransaction]Closing JDBC Connection [com.mysql.jdbc.JDBC4Connection@1efee8e7]
[DEBUG][21-03-06][org.apache.ibatis.datasource.pooled.PooledDataSource]Returned connection 520022247 to pool.
[INFO][21-03-06][class com.yi.dao.UserDaoTest]info:进入了testlog4j
[DEBUG][21-03-06][class com.yi.dao.UserDaoTest]debug:进入了testLog4j
[ERROR][21-03-06][class com.yi.dao.UserDaoTest]error:进入了testLog4j
